app/controllers/users_controller.rb

```diff
@@ -8,10 +8,11 @@ class UsersController < ApplicationController
   has_scope :tagged_with, as: :tag
 
   def index
-    members = current_organization.members.active
-    members = apply_scopes(members)
+  members = current_organization.members.active
+  members = apply_scopes(members)
 
-    search_and_load_members members, { s: 'user_last_sign_in_at DESC' }
+  # Pasamos directamente los parámetros sin forzar un orden en este punto
+  search_and_load_members(members, params[:q])
```

Comment:
you have removed identation here, it's importante to take care of the identation so it is more readable